src/Google.Api.Gax/project.json

```diff
@@ -17,16 +17,14 @@
   },
 
   "dependencies": {
-    "Google.Apis.Auth": "1.16.0",
-    "Google.Protobuf": "3.0.0",
-    "Google.Protobuf.Tools": { "version": "3.0.0", "type": "build" },
-    "Grpc.Auth": "1.0.1-pre1",
-    "Grpc.Core": "1.0.1-pre1",
-    "Grpc.Tools": { "version": "1.0.0", "type": "build" },
     "System.Interactive.Async": "3.0.0"
   },
   "frameworks": {
-    "net45": { },
-    "netstandard1.5": { }
+    "net45": {},
+    "netstandard1.3": {

```

Comment:
Nope, we can target 1.3 for Google.Api.Gax, which means we can keep targeting 1.3 for Google.Storage.V1 and Google.Bigquery.V1.

But Google.Api.Gax.Grpc still targets 1.5, because that's all that gRPC supports.